code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1173,7 +1173,8 @@
 	M.AdjustKnockdown(-20, 0)
 	M.AdjustUnconscious(-20, 0)
 	M.adjustStaminaLoss(-0.5*REM, 0)
-	..()
```

Comment:
er, don't you have to call ..() after everything?

Reply:
If you manually put in code to remove an amount of the reagent on every tick nope, holy water does this too IIRC.

Reply:
The base does almost the same, only difference is in the value, that takes in account the metabolism rate and efficiency